tests/e2e/specs/drawing.spec.js

```diff
@@ -0,0 +1,111 @@
+const mockResponse = { fileId: 'test', adminId: 'test' }
+
+describe('Drawing', () => {
+    function goToDrawing() {
+        cy.goToMapView()
+        cy.get('[data-cy="menu-button"]').click()
+        cy.get('.menu-section-head-title:first').click() // FIXME: how to address a specific menusection
+        cy.readStoreValue('state.ui.showDrawingOverlay').then((value) => {
+            expect(value).to.equal(true)
+        })
```

Comment:
can be simplified as
```suggestion
        cy.readStoreValue('state.ui.showDrawingOverlay').should('be.true')
```